### ISSUE TITLE
Add support for JOINs

### ISSUE THREAD

**Issue by otisg** [opbeat/sql-summary#3]
It seems only the first table is extracted, right?  Would it be possible to add support for JOINs?

**Comment by watson** [comment]
@otisg what did you imagine that the output would be? Something like `SELECT FROM foo, bar, baz` or?

**Comment by otisg** [comment]
@watson Yes, that's the sort of thing I was imagining.  Maybe with tables sorted in the same fashion all the time to avoid having both SELECT FROM foo, bar and SELECT FROM bar, foo.
Doable?

**Comment by watson** [comment]
@otisg It's a pretty complicated thing to parse given the complexity of SQL, so I don't know when I will have time to do this. In the meantime you are of cause welcome to submit a pull request :smiley:

**Comment by otisg** [comment]
Perhaps https://www.npmjs.com/package/node-sqlparser could be used?

**Comment by watson** [comment]
@otisg I was looking at it before making this module, but it's really really complicated (no critique... it has to because it parses the entire SQL query). I wanted a simpler module for a simpler purpose that wouldn't have that big an overhead. But there might be something in there I can use though - thanks for reminding me :smiley:
